odoh_server.go

```diff
@@ -116,15 +118,25 @@ func main() {
 	log.Printf("Setting Server Name as %v", serverName)
 
 	var experimentID string
-	if experimentID := os.Getenv(experimentIDEnvironmentVariable); experimentID == "" {
+	if experimentID = os.Getenv(experimentIDEnvironmentVariable); experimentID == "" {
 		experimentID = "EXP_LOCAL"
 	}
 
 	var telemetryType string
-	if telemetryType := os.Getenv(telemetryTypeEnvironmentVariable); telemetryType == "" {
+	if telemetryType = os.Getenv(telemetryTypeEnvironmentVariable); telemetryType == "" {
 		telemetryType = "LOG"
 	}
 
+	var certFile string
+	if certFile = os.Getenv(certificateEnvironmentVariable); certFile == "" {
+		certFile = "cert.pem"
+	}
+
+	var keyFile string
+	if keyFile = os.Getenv(keyEnvironmentVariable); keyFile == "" {
+		keyFile = "key.pem"
+	}
+
```

Comment:
You could possibly check if the `keyFile` and `certFile` exist with `os.Stat` (e.g., https://golangcode.com/check-if-a-file-exists/)

Reply:
Yeah, good to do in a followup issue. Can you please file an issue?

Reply:
Done: https://github.com/cloudflare/odoh-server-go/issues/17